src/decode.h

```diff
@@ -610,6 +610,9 @@ typedef struct Packet_
      */
     struct PktPool_ *pool;
 
+    /* make sure we have no too deep recursion */
```

Comment:
Would like to see something that explains more: "count layers of packet decoding, to avoid loops and too many layers cause issues with performance and stability (stack exhaustion)"

Reply:
ok